otx/algorithms/classification/tasks/inference.py

```diff
@@ -0,0 +1,391 @@
+"""Inference Task of OTX Classification."""
```

Comment:
where is nncf task for classification?

Reply:
In adapter deep-object-reid task. Just seperated it to avoid confusion.